notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>"in order to write a condition for it"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>